pkg/vm/ch.go

```diff
@@ -17,9 +19,34 @@ import (
 )
 
 const (
-	chBin = "cloud-hypervisor"
+	chBin           = "cloud-hypervisor"
+	cloudConsoleBin = "cloud-console"
 )
 
+func (m *Machine) StartCloudConsole(ctx context.Context, namespace string, networkAddr net.IPNet, machineIP net.IPNet, ptyPath string) error {
+	machineIPLsit := strings.Split(machineIP.IP.String(), ".")
+	port, err := strconv.Atoi(machineIPLsit[len(machineIPLsit)-1])
+	if err != nil {
+		return err
+	}
+	port = 20000 + port
```

Comment:
where does the 20000 number come from? and is there a guard against max port used 65535?